problem/validate-parenthesis_en.md

```diff
@@ -1,4 +1,47 @@
+# Valid Parentheses Checker
 
-There is no solution yet.
+This code implements a solution to check if a given string containing different types of brackets is valid or not. It uses a stack data structure to maintain the order of opening and closing brackets and ensure their validity.
 
-Would you like to [contribute to the solution](https://github.com/BFEdev/BFE.dev-solutions/blob/main/problem/validate-parenthesis_en.md)? [Contribute guideline](https://github.com/BFEdev/BFE.dev-solutions#how-to-contribute)
+You can watch a video explanation of this solution [here](https://youtu.be/lwN4SDGTBaY?si=HTgiwVLHvQjaxMuK).
```

Comment:
You can add your github link but not links to other platforms
other than this, it looks good

Reply:
thx, updated link